client/src/templates/Challenges/classic/action-row.tsx

```diff
@@ -38,36 +38,35 @@ const ActionRow = ({
       </div>
       <div className='tabs-row'>
         <EditorTabs />
-        <button
-          className='restart-step-tab'
-          onClick={resetChallenge}
-          role='tab'
-        >
+        <button className='restart-step-tab' onClick={resetChallenge}>
           {t('learn.editor-tabs.restart-step')}
         </button>
         <div className='panel-display-tabs'>
           <button
+            aria-expanded={showConsole ? 'true' : 'false'}
             className={showConsole ? 'active-tab' : ''}
             onClick={() => togglePane('showConsole')}
-            role='tab'
           >
-            {t('learn.editor-tabs.console')}
+            {t('learn.editor-tabs.console')}{' '}
+            <span className='sr-only'>display</span>
```

Comment:
We will need to translate the word "display" too, I think.

To help provide the translators enough context, my suggestion is to have two set of text, say `learn.editor-tabs` and `learn.editor-tabs-sr`. We would then set `sr-only` to the former.

But would love to hear your thoughts as well.